script/clone_job.pl

```diff
@@ -301,7 +301,7 @@ sub clone_job {
         }
     }
     else {
-        die "failed to create job: ", pp($tx->res->body);
+        die "failed to create job: please make sure your http proxy is running (apache, nginx, etc.) ", pp($tx->res->body);
```

Comment:
@okurz I just got the line where the issue occured ("failed to create job: "" at ./script/clone_job.pl line 304.")